core/primitives/src/sr25519.rs

```diff
@@ -379,6 +379,7 @@ fn derive_hard_junction(secret: &SecretKey, cc: &[u8; CHAIN_CODE_LENGTH]) -> Sec
 	secret.hard_derive_mini_secret_key(Some(ChainCode(cc.clone())), b"").0.expand()
 }
 
+/// The raw `MiniSecretKey` bytes.
```

Comment:
I don't understnad why it is called in the docs `MinitSecretKey` and as type name `Seed`?

Reply:
I can actually rollback that change, cause at the end I ended up keeping `Seed` private and just used the recovery phrase to store it in the database.

But my understanding is that `Seed` is a thing that is used to actually create the `Pair` type (via `Pair::from_seed`).
And for `sr25519` this happens to be the `MiniSecretKey`:
```rust
// This is generated using schnorrkel's Mini-Secret-Keys.                          
///                                                                                 
/// A MiniSecretKey is literally what Ed25519 calls a SecretKey, which is just 32 random bytes.   
```
hence the comment.

Reply:
Prefer `Seed` or the seed phrase to `MiniSecretKey` or `SecretKey` - the latter two are not always available for effective access or are specific to particular crypto systems.

Reply:
"Seed" is just a crypto-generic concept for some datatype that when filled with random bytes can uniquely generate a "secret key". Some cryptos have restrictions on their secret keys, some crypto libraries don't let you access the secret key. So this is a useful abstraction.